src/main/resources/assets/js/components/AppHealthComponent.jsx

```diff
@@ -0,0 +1,78 @@
+/** @jsx React.DOM */
+
+var React = require("react/addons");
+
+var AppHealthComponent = React.createClass({
+  displayName: "AppHealthComponent",
+
+  getHealthData: function () {
+    var model = this.props.model;
+
+    var tasksWithUnknownHealth = Math.max(
+      model.get("tasksRunning") -
+      model.get("tasksHealthy") -
+      model.get("tasksUnhealthy"),
+      0
+    );
+
+    var healthData = [
+      {quantity: model.get("tasksHealthy"), className: "health-bar-healthy"},
+      {quantity: model.get("tasksUnhealthy"), className: "health-bar-unhealthy"}, /* jscs:disable maximumLineLength */
```

Comment:
I think it is possible make this look good without breaking the 80 character rule. Maybe it would be it would be an idea to create variables for the `task.get("...");` that you use multiple times? I think that would make it more readable, what are your thoughts?